compiler/circle-opselector/src/TestHelper.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __CIRCLE2CIRCLE_TEST_HELPER_H__
+#define __CIRCLE2CIRCLE_TEST_HELPER_H__
+
+#include <luci/IR/Module.h>
+#include "ModuleIO.h"
+
+#include <cassert>
+#include <cstdio>
+#include <vector>
+#include <string.h>
+#include <string>
+
+int entry(int argc, char **argv);
+bool is_number(const std::string &s);
+std::vector<uint32_t> split_id_input(const std::string &str);
```

Comment:
I think these would be to move inside `Driver.test.cpp` or introduce into `Driver.test.h`